src/messages/responses/list-caches/list-caches.ts

```diff
@@ -0,0 +1,54 @@
+import {ListCachesResponse} from './list-caches-response';
+import {CacheInfo} from '../../cache-info';
+import {control} from '@gomomento/generated-types';
+import {MomentoErrorCode, SdkError} from '../../../errors/errors';
+
+export class Success extends ListCachesResponse {
+  private readonly nextToken: string | null;
+  private readonly caches: CacheInfo[];
+  constructor(result?: control.control_client._ListCachesResponse) {
+    super();
+    this.nextToken = result?.next_token || null;
+    this.caches = [];
+    result?.cache.forEach(cache =>
+      this.caches.push(new CacheInfo(cache.cache_name))
+    );
+  }
+
+  public getNextToken() {
+    return this.nextToken;
+  }
+
+  public getCaches() {
+    return this.caches;
+  }
```

Comment:
The Error classes use an public attribute() / private _attribute` pattern. This uses a public getAttribute() / private attribute` pattern. Would it be better to make them consistent?

Reply:
Yes, good call. I'll update the errors.